review/tests.py

```diff
@@ -51,3 +51,21 @@ def test_delete_related_dogowner(self, test_review):
         dogowner.delete()
         assert dogowner not in DogOwner.objects.all()
         assert test_review in Review.objects.all()
+
+    def test_get_review_by_daycare_id(self):
+        daycare = DayCare.create(email="review@gmail.com", username="review", password="pass", name="reviewDayCare",
```

Comment:
Unless there's something special about this daycare, we may want to have a daycare fixture under contest on project-level.

Reply:
changed